src/resources/views/crud/filters/date_range.blade.php

```diff
@@ -1,21 +1,28 @@
 {{-- Date Range Backpack CRUD filter --}}
 
 @php
-    $filter->options['date_range_options'] = array_merge([
+    $filter_language = $filter->options['date_range_options']['language'] ?? app()->getLocale();
```

Comment:
@pxpm does it makes sense to expect a language in this `date_range` field, since it isn't being done in any other one?
Or should we just assume it will always be `app()->getLocale()`?

Reply:
The way I see it, we are adding this options to `date_range` in filter, but the plan is to make it available to the field too. I think it makes more sense in field area, it's not expensive from maintainability side and who knows what kind of app that people are building that could not have the same fields with both languages in one crud. 

`app()->getLocale()` is a sensible default, but still our default, so allowing for override when it's not expensive for us in my opinion is best to have than to not. Anyway it will fallback nicely to app locale, so only people that would really need this feature would use it.

Reply:
Get it, yeah, we never know what people do 😅
I'm thinking about a solution, that's not very elegant, but it would help out a lot with maintenance of this feature.

If we really want columns/fields/filters to have specific languages, what about "overriding" App::locale temporary while rendering each columns/fields/filters?

For columns, for instance, on `renderCellView` we would check if user provided a `"language"` attribute, if so, we could temporary change App::locale, render the view, and then go back to previous locale.

**PROS**:
- This small change could provided this feature for all columns/fields/filters instantly.
- No messy code on columns/fields/filters to deal with user selected language, and translate texts with it.
- No maintenance is necessary, it just works.
- No need to refactor dozens of blades 😅
- Not braking change.

**CONS**:
- It's not elegant, messing with this global vars 🤷‍♂️

What do you think @pxpm?
Maybe @tabacitu should give his opinion on this too.

Reply:
I agree that the best solution would be to have the same functionality accross all fields/columns etc, but also is important for people that does not want it to be able to disable (or people who want be able to enable).

There is also other request that we are willing to provide that is when you are editing an entry in some other language than the default and use for example a `select` field, that the select options should be present in the editing language instead of the app language. #3228

Reply:
Just talked to Pedro. We decided that the best way to go forward would be to _use the current locale_, instead of allowing the developer to change the locale.

But the developer will be able to change this filter language, by changing the entire admin panel language, in a global dropdown, like suggested in https://github.com/Laravel-Backpack/ideas/issues/93

Additionally, for the date_range FIELD (!!!) we should:
- include moment with locales so that it will automatically translate into the current language;
- add a config to the `Update` operation, say `use_editing_language_instead_of_app_locale` (boolean) that would do exactly what the name says; then the developer would be able to toggle that inside `config/backpack/crud.php` or inside a particular CRUD controller, and ALL the fields would be output using the chosen Language; see https://github.com/Laravel-Backpack/CRUD/issues/3228#issuecomment-720564155 but basically it would imply overwriting the app_locale() like @promatik said before & after the form, most likely inside `form_content.blade.php`;

Reply:
We can do those in 4.1.x, and merge that with this one at the same time (after changing so that the language is no longer configurable), and we'll have a complete solution 🥳